src/main/resources/templates/startPage.html

```diff
@@ -12,23 +12,28 @@
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.1.3/dist/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
 </head>
 <h1>Simple Talk</h1>
-<body>
-    <br />
+<body style="padding: 20px 40px">
     <form action="#"  method="GET">
         <label for="searchTerm">Search</label><input type="text" name="searchTerm" id="searchTerm"/>
-        <button name="submit" tyoe="submit" value="0">Search</button>
+        <button class="btn" name="submit" type="submit" value="0">Search</button>
     </form>
-    <br />
+    <br>
```

Comment:
Keep this as is.

br / is proper XHMTL (XML and HTML compliant)
br is only HTML compliant.  
Shoot for XHTML compliance whenever possible.